### ISSUE TITLE
Cannot keep modifications to cron

### ISSUE THREAD

**Issue by Germar** [bit-team/backintime#105]
I have made two modifications to cron via gnome-schedule:

1. To run on Mondays (00**1) instead of Sundays (00**0), and
2. To print output: /usr/bin/backintime --backup-job  >> /home/user/Desktop/cron.log

But whenever I run backintime gui, cron goes back to the unmodified setting.

Is there any way to stop this behaviour? 

Thanks,

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2011-01-06T13:15:21Z
 * owner: jsmm
 * assignee: germar
 * the launchpad url was https://bugs.launchpad.net/bugs/698106

**Comment by Germar** [comment]
(by danleweb)
For now every time you configure BIT it remove all "backintime" crontab entries. And this is a bug.

You can create a script that run "backintime --backup" and call it something like bit.sh. That add it manually.


Regards,
Dan

**Comment by Germar** [comment]
(by profpatsch)
How about adding "#BACKINTIME_AUTO" to the end of every automatically created line and only removing them?
Can’t be too hard, can it?

I have the same problem, I want to use fcron’s uptime-dependent options but they get deleted by the gui. I suppose I have to write a patch myself.

**Comment by Germar** [comment]
(by profpatsch)
Wrote a simple solution.

When there is nothing to write, appends the following to lines to the cron fie:
#Back In Time system entry, this will be edited by the gui:
#Please don't delete these two lines, or all backintime. entries are going to be deleted next time you call the gui options!"

First one is the system_entry_message and the second one contains backintime., meaning it will be removed next time together with the first line.

You are going to have to edit the indentation, because WHAT THE HECK you are mixing tabs and spaces! Seriously?
All I can say is that my vim didn’t want to edit these without me fiddling with my config files.

Anyways, you’re welcome.

**Comment by Germar** [comment]
(by germar)
Thanks for the patch. I'll take a look on it soon.

You're right about the indentation issue. I already talked about this with Dan last October and we decided to fix this after the 1.0.12 release. But then we run into trouble with the PPA with that release. So we delayed it again. At the moment I've one branch (~bit-team/backintime/password ) that I want to finish and merge before addressing the indentation issue. But it will be done :)
